### PR TITLE
B opt out photo

### PR DESCRIPTION
**Pull request by Doug-Reed**
This changeset provides the functionality to allow a user to opt out of using their photo as an avatar. (The photo functionality isn't built yet.)

If a user is in a theme which allows an avatar, (as specified by putting an avatar value in uw-frame-components/js/frame-config.js), and the user has toggled yes to "Allow use of my photo" in /settings, then that user will currently see the word "PIC". This is a placeholder until the photo functionality is built. 

The user can toggle off the switch and opt out. 
----

Contributor License Agreement adherence:

- [ X] This Contribution is under the terms of Individual [Contributor License Agreements][] (and also Corporate Contributor License Agreements to the extent applicable) appearing in the [Apereo CLA roster][].

[Apereo CLA roster]: http://licensing.apereo.org/completed-clas
[Contributor License Agreements]: https://www.apereo.org/licensing/agreements

### CODE REVIEW

File: uw-frame-components/js/frame-config.js [hunk 1]
```diff
@@ -1069,5 +1070,11 @@ define(['angular'], function(angular) {
             'description': 'This flag disables announcement group filtering ' +
               'if you have it enabled (page refresh required)',
           },
+          {
+            'id': 'optInAvatar',
+            'title': 'Allow Use Of My Photo',
+            'description': 'This flag allows use of your photograph ' +
+              'if you have it enabled (page refresh required)',
+          },
```
Comment (davidmsibley) [inline uw-frame-components/js/frame-config.js]: This method of opting in would require the user to know about and navigate to our nuts and bolts settings page. We should keep this feature flag, but maybe add a toggle in the user settings area?

Comment (apetro) [inline uw-frame-components/js/frame-config.js]: Agree. Would be best for the user-facing opt-in / opt-out setting to be in the `/user-settings` page. Could hide the presentation of that setting behind its own feature flag, but probably better to just hide it behind whether the active theme supports avatar image feature.

**Comment by Doug-Reed** [comment]
I'll move that toggle setting.

**Comment by davidmsibley** [comment]
Can this be closed since #469 was merged?

**Pull request closed without merging**